VERSION

```diff
@@ -1 +1 @@
-2.13.0
\ No newline at end of file
+2.13.1
```

Comment:
Shouldn't this have a newline at the end? This one is the only VERSION file in any Prometheus project without a trailing newline.

Reply:
I'm guessing it is an artifact of https://github.com/prometheus/promu/pull/170

Reply:
I'd suggest to fix it in `promu bump` then.

Reply:
In progress though `promu bump` doesn't exist yet ;-)

Reply:
> Thinking about #6139 : Perhaps it will help to bump prometheus/client_golang to v1.2.0.

IIUC the patch release should add only include bug fixes introduced since the last version right? 
Is this one of those or has been there before that?